tests/DependencyInjection/Compiler/AddDependencyCallsCompilerPassTest.php

```diff
@@ -681,167 +602,75 @@ protected function getConfig()
         ];
     }
 
-    private function getContainer(): ContainerBuilder
+    protected function registerCompilerPass(ContainerBuilder $container): void
     {
-        $container = new ContainerBuilder();
-        $container->setParameter('kernel.bundles', [
+        $container->addCompilerPass(new AddDependencyCallsCompilerPass());
+    }
+
+    private function setUpContainer(): void
+    {
+        $this->container->setParameter('kernel.bundles', [
+            'SonataCoreBundle' => true,
```

Comment:
oops just saw this, can you please remove it?

Reply:
Sure, done